types/validator/index.d.ts

```diff
@@ -317,6 +317,11 @@ declare namespace validator {
      */
     function isDataURI(str: string): boolean;
 
+    /**
+     * Check if the string is a valid date.
+     */
+    function isDate(str: string): boolean;
```

Comment:
`isDate` accepts also second `options` parameter (according to docs) but I believe this change is not released yet.

Reply:
I missed that, I have now updated the code to work with the options parameter.

Reply:
I think we should wait while `13.1.17` does not correspond with this `isDate` type definition yet. I think it will be part of next release of validatorjs.